### DISCUSSION TITLE
¿Is it for Win 7?

### DISCUSSION THREAD

**Discussion by jazei-AR** [qutebrowser/qutebrowser#6625] [Q&A] [answered]
I red from https://www.softpedia.com/get/Internet/Browsers/qutebrowser.shtml that  it is for win 7...
![Sin título2](https://user-images.githubusercontent.com/72522918/127382011-2eecbc3f-8db7-4e7f-97b7-f6b9f82782f6.jpg)
so I dowloaded and get this message:
![Sin título](https://user-images.githubusercontent.com/72522918/127382077-6cf0487f-fd5f-4877-8eaa-71e9bcb810b7.jpg)

**Comment by The-Compiler** [comment, marked as answer]
The setup is correct, Softpedia is wrong. Support for Windows 7 [has been dropped](https://github.com/qutebrowser/qutebrowser/blob/master/doc/changelog.asciidoc#v200-2021-01-28) in v2.0.0, given that Microsoft's support for Windows 7 ended in January 2020.

**Reply by jazei-AR** [reply]
thanks and good bye

**Reply by mschilli87** [reply]
@jazei-AR: But I guess if you are a fan of outdated software and do not care about security at all, you could probably just run [an old version of qutebrowser](https://github.com/qutebrowser/qutebrowser/releases/tag/v1.14.1). :wink:
